modules/acquia_cms_common/tests/src/ExistingSite/BreadcrumbTest.php

```diff
@@ -162,15 +162,15 @@ private function assertBreadcrumb(array $expected_breadcrumb) : void {
       if ($link) {
         return [
           $link->getText(),
-          $link->getAttribute('href'),
+          // Let remove -tid from the link generated by facet pretty path.
+          preg_replace('/-[^-]*$/', '', $link->getAttribute('href')),
```

Comment:
@chandan-singh7929 Isin't this would return wrong url, if URL contains multiple hyphens. 
Ex: /meet-up/something-123. This would return => /meet
cc: @gauravgoyal

Reply:
@vishalkhode1 - I have tested this, it will only remove string from last part of the URL in this case(`something-123`), not from the beginning or middle